exasol/nb_connector/extension_wrapper_common.py

```diff
@@ -73,16 +75,21 @@ def deploy_language_container(conf: Secrets,
         conf:
             The secret store. The store must contain the DB connection parameters
             and the parameters of the BucketFS service.
-        container_url:
-            The url to download the language container from
-        container_name:
-            The language container will be saved in the BucketFS with this name.
         path_in_bucket:
             Path in the BucketFS where the container should be saved.
         language_alias:
             The language alias of the extension's language container.
         activation_key:
             A secret store key for saving the activation SQL.
+        container_url:
+            An optional url to download the language container from.
+            Either the `container_url` or `container_file` should be provided.
+        container_file:
+            An optional path of the container file (*.tar.gz) in a local file system.
+            Either the `container_url` or `container_file` should be provided.
+        container_name:
+            If provided, the language container will be saved in the BucketFS with
+            this name. Otherwise, the name of the container file will be used.
```

Comment:
```suggestion
            If provided, the language container will be saved in given bucket of BucketFS with
            this filename. Otherwise, the name of the container file will be used.
```